packages/arb-token-bridge-ui/src/util/withdrawals/fetchWithdrawalsFromSubgraph.ts

```diff
@@ -11,9 +11,11 @@ export type FetchWithdrawalsFromSubgraphResult = {
   }
   tokenAmount: string
   isClassic: boolean
-  l2BlockTimestamp: string
+  timestamp: string
   l2TxHash: string
   l2BlockNum: string
+  parentChainId: number
+  chainId: number
```

Comment:
Let's do `childChainId` everywhere. Christophe's `useNetworksRelationship` hook will give us `parentChain` and `childChain`

Reply:
done